docs/csharp/programming-guide/interop/walkthrough-office-programming.md

```diff
@@ -199,11 +199,7 @@ You must have Microsoft Office Excel and Microsoft Office Word installed on your
 - [Lambda Expressions (C#)](../statements-expressions-operators/lambda-expressions.md)
 - [How to: Use Indexed Properties in COM Interop Programming](./how-to-use-indexed-properties-in-com-interop-rogramming.md)
 - [Walkthrough: Embedding Type Information from Microsoft Office Assemblies in Visual Studio](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120))
-<<<<<<< HEAD
```

Comment:
ouch... let me double check we don't have more of those in our docs after that giant PR was merged.

Reply:
I did a quick search for `<<<<` and that was the only one I found
Feel free to land/fix this sepately if you want to just fix the page

Reply:
That's good. Thanks for checking!